src/main/java/com/squareup/kotlinpoet/AnnotationSpec.kt

```diff
@@ -30,6 +30,25 @@ class AnnotationSpec private constructor(builder: AnnotationSpec.Builder) {
   val className: ClassName = builder.className
   val members = builder.members.toImmutableList()
   val useSiteTarget: UseSiteTarget? = builder.useSiteTarget
+  private val tags: Map<KClass<*>, Any> = builder.tags.toImmutableMap()
+
+  /**
+   * Returns the tag attached with [Any] as a key, or `null` if no tag is attached with that key.
+   */
+  fun tag(): Any? = tag(Any::class)
```

Comment:
I don't think we need this variant nor the builder method. Its use breaks composition of tags which seems like the only reason to have them given they're public API.

Same comment on all specs

Reply:
2c91a6b